### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by auanasgheps**
- reorganize strings
- add new tranlsations

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -139,6 +145,12 @@
     <string name="settings_update_beta">Beta</string>
     <string name="settings_update_custom">Canale personalizzato</string>
     <string name="settings_update_custom_msg">Inserisci un URL personalizzato</string>
+    <string name="settings_zygisk_summary">Esegui parti di Magisk nel processo zygote</string>
+	<string name="settings_denylist_title">Applica lista di blocco</string>
+	<string name="settings_denylist_summary">Tutte le modifiche Magisk verranno annullate per i processi in lista di blocco</string>
```
Comment (Fs00) [inline app/src/main/res/values-it/strings.xml:150]:
```suggestion
	<string name="settings_denylist_summary">Tutte le modifiche apportate da Magisk verranno annullate per i processi in lista di blocco</string>
```

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -139,6 +145,12 @@
...
+    <string name="settings_denylist_error">Questa funzione richiede %1$s abilitato</string>
```
Comment (Fs00) [inline app/src/main/res/values-it/strings.xml:151]:
```suggestion
    <string name="settings_denylist_error">Questa funzione richiede che %1$s sia abilitato</string>
```

File: app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -139,6 +145,12 @@
...
+    <string name="settings_denylist_config_title">Configura lista di blocco</string>
+    <string name="settings_denylist_config_summary">Seleziona i processi da incluidere nella lista di blocco</string>
```
Comment (Fs00) [inline app/src/main/res/values-it/strings.xml:153]:
```suggestion
    <string name="settings_denylist_config_summary">Seleziona i processi da includere nella lista di blocco</string>
```

**Review by Fs00 — COMMENTED** [review]
Thank you! I've found only a couple of minor errors/improvable sentences.

**Comment by auanasgheps** [comment]
Looks good thanks for reporting!

**Review by topjohnwu — APPROVED** [review]

**Pull request merged**